README.md

```diff
@@ -350,6 +350,8 @@ If you want to uninstall algorithms, it is as simple as:
         - [simplify_path](algorithms/unix/path/simplify_path.py)
 - [union-find](algorithms/union-find)
     - [count_islands](algorithms/union-find/count_islands.py)
+- [machine-learning](algorithms/machine-learning)
```

Comment:
lets change the folder name to ml for simplicity.
As a package, we need to think how people are going to use it.
typing `algorithms.ml` is better than typing `algorithms.machine-learning` every time.

Reply:
@keon Good point! I rename the directory